tests/test_nodestats_regression.py

```diff
@@ -0,0 +1,57 @@
+"""Regression test for nodestats tracing."""
+
+import importlib.resources as pkg_resources
+from pathlib import Path
+
+import h5py
+import yaml
+from test_io import dict_almost_equal
+
+import topostats
+from topostats.io import LoadScans, hdf5_to_dict, read_yaml
+from topostats.processing import (
+    process_scan,
+)
+
+BASE_DIR = Path.cwd()
+REGRESSION = BASE_DIR / "tests/nodestats-regression"
```

Comment:
In light of above comment on `minicircle.topostats.out` this should be...

```suggestion
RESOURCES = BASE_DIR / "tests/resources"
```